autoload/vim_dadbod_completion/reserved_keywords.vim

```diff
@@ -147,7 +147,11 @@ let s:reserved_words = [
       \ 'WRITETEXT', 'X509', 'XOR', 'YEAR', 'YEAR_MONTH', 'ZEROFILL', 'ZONE',
       \ ]
 
```

Comment:
Could we just check the global variable and map the whole `s:reserved_words` to lowercase? that way we would do it only once, and there's no need to constantly map it.

Reply:
Absolutely, we would only need to additionally account for that change in
`autoload/vim_dadbod_completion/schemas.vim`

```vim
let s:quote_rules = {
      \ 'camelcase': {val -> val =~# '[A-Z]' && val =~# '[a-z]'},
      \ 'space': {val -> val =~# '\s'},
      \ 'reserved_word': {val -> has_key(s:reserved_words, toupper(val))}
      \ }
```
Since the `reserved_word` lambda assumes the reserved keywords are in upper case.